etc/zsh/_bloop

```diff
@@ -25,30 +27,53 @@ _boolean() {
 }
 
 _testsfqcn() {
-    project="$words[2]"
+    local project="$words[2]"
     compadd "$@" $(bloop autocomplete --format zsh --mode testsfqcn --project $project)
 }
 
 _mainsfqcn() {
-    project="$words[2]"
+    local project="$words[2]"
     compadd "$@" $(bloop autocomplete --format zsh --mode mainsfqcn --project $project)
 }
 
 _flags() {
-    cmd="$words[1]"
-    saveIFS=$IFS
+    local cmd="$words[1]"
+    _flags_for_cmd $cmd
+}
+
+_flags_for_cmd() {
+    local cmd=$1
+    local saveIFS=$IFS
     IFS=$'\n'
-    results=($(bloop autocomplete --format zsh --mode flags --command $cmd))
-    # Add a dummy result without completion, otherwise no completion is shown.
-    results=($results '*:dummy:')
+    local flags=($(bloop autocomplete --format zsh --mode flags --command $cmd))
+    flags=($flags ':approximate-4:::' )
     IFS=$saveIFS
+    _arguments $flags
+    return 0
+}
 
-    _arguments $results
+_project_or_flags() {
+    local project_bound_cmd=( clean compile run test console )
```

Comment:
Would it be possible to get the list of commands that take a project from `bloop autocomplete` too? :smile: